src/components/UserActions/UserActions.ts

```diff
@@ -231,6 +260,30 @@ export class UserActions extends Component {
         this.element.innerHTML = '';
         this.element.appendChild(element);
     }
+
+    private showViewedByCustomer() {
+        this.bind.onRootElement(ResultListEvents.newResultDisplayed, (args: IDisplayedNewResultEventArgs) => {
+            if (Boolean(args.item.getElementsByClassName('CoveoViewedByCustomer').length)) {
+                return;
+            }
+            const viewedByCustomerElement = document.createElement('span');
+            new ViewedByCustomer(viewedByCustomerElement, undefined, this.bindings, args.result);
+            const resultLastRow = '.coveo-result-row:last-child .coveo-result-cell';
+            args.item.querySelector(resultLastRow).appendChild(viewedByCustomerElement);
+        });
+    }
+
+    private tagViewsOfUser() {
+        Coveo.$$(this.root).on('buildingQuery', (e, args) => {
+            try {
+                args.queryBuilder.userActions = {
+                    tagViewsOfUser: this.options.createdBy
+                };
+            } catch (e) {
+                this.logger.warn("CreatedBy Email wasn't found", e);
```

Comment:
Maybe make stephanie review this error message 😄

Reply:
No need to review console error messages, but I agree that it shouldn't say "createdby.email" :P